templates/tutorialv2/validation/index.html

```diff
@@ -101,6 +102,13 @@ <h2 class="subtitle">{{ headlinesub|safe }}</h2>
                                 <td>
                                     <span>{{ validation.date_proposition|format_date:True|capfirst }}</span>
                                 </td>
+                                <td class="{% if not validation.content.in_public %}new_content{% else %}update_content{% endif %}">
+                                    {% if validation.content.is_public %}
+                                        {% trans "Nouveau" %}
```

Comment:
Les deux lignes 
```HTML+Django
{% if not validation.content.in_public %}new_content
```
et
```HTML+Django
{% if validation.content.is_public %}
    {% trans "Nouveau" %}
```
se contredisent, et la seconde semble fausse (il faudrait `{% if not ... %}` partout).